### ISSUE TITLE
Run all tests in a folder

### ISSUE THREAD

**Issue by simonalexander2005** [zheng-wang/irontest#20]
In order to improve the usefulness of the tool, it would be helpful to be able to run all tests in a folder without having to open and run each individually; and then be able to see a report that covers all the tests in the folder. This view could perhaps happen when the folder name is left-clicked in the UI. That view could also show a summary of when each test was last run and whether it passed or not.

**Comment by zheng-wang** [comment]
This feature is definitely required. Actually it has been in my backlog for long time due to
* its complexity to implement
* its relatively lower value comparing to other improvements to do (based on my own work experience)
* my time limitation

The conditions above still exist, so please be patient on this feature's arrival time. :)

**Comment by simonalexander2005** [comment]
Yes of course. It would be useful if it had the option to run in parallel (if the tests don't interact with each other), or in sequence (if they would otherwise interfere with each other's running)

**Comment by zheng-wang** [comment]
Right, besides parallel/sequence running of test cases in one folder, there is also consideration as to parallel/sequence running of different folders (run one folder, then quickly go to another folder and run it). Will need to go into more details when starting to implement this feature.

**Comment by simonalexander2005** [comment]
Yes, and to have Properties that can be set at folder-level too

**Comment by zheng-wang** [comment]
Properties on folder is a different feature, so need another issue to discuss. 

Actually I am sure you are able to open another 1000 issues to Iron Test given it is far from version 1.0 and would probably need a couple of years to get there. : )

So my suggestion is for you to open issues that have big value to your current work, so that my limited effort can effectively benefit you and hopefully benefit the community as well.
